closed/adds/jdk/src/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -50,18 +50,17 @@ public class NativeCrypto {
     private static final boolean traceEnabled = Boolean.parseBoolean(
             GetPropertyAction.privilegedGetProperty("jdk.nativeCryptoTrace", "false"));
 
-    //ossl_ver:
+    //ossl_vers will be either:
     // -1 : library load failed
-    //  0 : openssl 1.0.x
-    //  1 : openssl 1.1.x or newer
+    // or one of the OPENSSL_VERSION_x_x_x constants
```

Comment:
Why are those "OPENSSL_VERSION_x_x_x constants" not declared here?
Please add them or update the comment to make it clear those constants are declared in `NativeCrypto.c`.

Reply:
Added.